test/unittests/rest/RestService_test.cpp

```diff
@@ -138,7 +138,7 @@ TEST(RestService, payloadParse)
 /* ****************************************************************************
 *
 * noSuchService -
-*/
+*//*
```

Comment:
This seems to be pending... only the present() calls should be removed

Reply:
fixed in d3c393e